<launch>
	<node pkg="mavros" name="mavros_node" type="pkg="mavros_node"" args="$(_fcu_url tcp://127.0.0.1:5760) $(_system_id 2)" />
	<node pkg="px4_quad_controllers" name="attitude_thrust_controller" type="attitude_thrust_controller.py" output="screen">
	<rosparam file="$(find px4_quad_controllers)/param/params.yaml" />
	</node>

	<node pkg="px4_quad_controllers" name="hover_height_position_controller" type="hover_height_position_controller.py" output="screen"/>
</launch>
